publish-github-release/index.js

```diff
@@ -94,6 +99,42 @@ async function run()
         }).catch(err => {
             console.log(`Error: ${err.message}`);
         });
+        const modulesJsonPath = "modules_v3.json";
```

Comment:
not sure but let me suggest to define modulesJsonPath as an input parameter for the action
This will allow to customize workflow for different versions v4, v5, ...

Reply:
It is the name of a temporary file that will be deleted after the end of the workflow